### ISSUE TITLE
I can't install poetry in Ubuntu 22.04 version

### ISSUE THREAD

**Issue by danger112233** [python-poetry/poetry#8527]
Hi Team,

When I try to install poetry I facing this type of issue 
![IMG-20231009-WA0009](https://github.com/python-poetry/poetry/assets/63256615/49f509c1-ae31-4bec-aa2a-046cdeb9178a)

Machine : Ubuntu 22.04 
I have attached screenshot also. Please guide me on this..

Regards,
Askar

**Comment by danger112233** [comment]
I'm completely new to this if made a mistake kindly give step by step instructions.. when I read a documentation it completely confusing..

**Comment by clintonroy** [comment]
Never post screenshots of text, always copy and paste the text. Always attempt to follow the bug template and fill it out.

If you're on ubutnu, you should be able to apt install pipx, then i'd install poetry with pipx install poetry.

**Comment by swills1** [comment]
Hello,

As @clintonroy pointed out, it's bad form to post a screenshot instead of the text. It makes more work for the people trying to help.

Something to know is - not being able to install is almost always going to be specific to the person instlling. The odds of poetry not being able to install across the board for everyone everywhere would mean some kind of major outage. Which is very unlikely.

When having trouble installing a package, the project Issues section isn't the best place to ask for help. It's better to ask in a support community such as Reddit, Stack Overflow, or my favorite - the Python Discord server. You may try https://askubuntu.com/.

As far as not being able to install on Ubuntu 22.04 -

I just created a new VM and installed Ubuntu 22.04.03. Once the OS was installed I did the following;
1. sudo apt update -y
2. sudo apt upgrade -y
* Steps 1 an 2 gave me an up to date VM to act as my baseline.
* So now I had a fresh and up to date Ubuntu 22.04.03 intall.
3. I ran `python3 --version` which showed me my version. `Python 3.10.12`

Every guide I looked up said to install using `curl -sSL https://install.python-poetry.org | python3 -`.

Then do a `poetry --version`

This resulted inan error. So I tried to install via pip `python3 -m pip install poetry`. I tried to use Poetry and I got errors. I even installed via the distro package `sudo apt install python3-poetry.

I modified the bashrc and I rebooted. Nothing worked. Now I know if i go down the rabbit-hole far enough, I will find steps that work, but I don't need to. I can say with absolute certainty this is an Ubuntu issue and not a Poetry issue. My development environemnt is entirely Fedora and I have never had this level of trouble. I install Fedora, I update everything, I run `python -m pip install poetry` and from there I can crate a new project, go into the shell, and work. No hurdles.

From installing the Ubuntu VM above and playing with it, I can say Ubuntu is just handling things in a very strange way. This does not happen on other distros. You should create a thread at or search https://askubuntu.com/.

**Comment by dimbleby** [comment]
> kindly give step by step instructions

@danger112233 there are step by step instructions on the poetry web page, please follow them

> The odds of poetry not being able to install across the board for everyone everywhere would mean some kind of major outage. Which is very unlikely

@swills1 the same is true of the odds of poetry not being able to install in ubuntu 22.04.  Indeed poetry's own pipelines use ubuntu 22.04, there is no problem.

If you are able to reproduce a problem, suggest showing that in a Dockerfile so that everyone else can reproduce it too.  Here is a starter:
```dockerfile
FROM ubuntu:22.04

RUN apt-get update && \
    apt-get upgrade -y && \
    apt-get install \
      --yes \
      --no-install-recommends \
      ca-certificates curl python3 && \
    curl -sSL https://install.python-poetry.org | python3 -
```

Otherwise I expect there's nothing to be done here, just some combination of user error and not reading the docs.

**Comment by swills1** [comment]
@dimbleby I didn't share a Docker file because it would just exacerbate this open issue. I mentioned above how I could go all the way down the rabbit-hold, but that would just take me on an un-needed journey. The point of my reply was more so was to have someone who uses Poetry daily but isn't a Poetry developer say - This is not a Poetry issue.

I just spun up a fresh Ubutu 22.04 VM on my hypervisor and perfrmed the following. These steps led to a successful install. Anyone reading should be able to follow and install. And hopefully with htat, this can be closed.

I like to do my `update` and `upgrade` separately.

1. sudo apt uppdate -y
2. sudo apt upgrade -y
3. sudo apt install vim
4. vim ~/.bashrc
5. Go to bottom of bashrc
6. Add `export PATH="$HOME/.poetry/bin:$PATH"`
7. I did `source ~/.bashrc` but couldn't run `poetry --version` So I just rebooted and Poetry ws working fine. 

In my last test, I just didn't reboot to reload the bachrc thinking `source` had done it.

Poetry is working fine on Ubuntu 22.04 for me and should for anyone who follows those steps above.

Thanks, all, and thanks for Poetry.
